feature/experimental/p4rt/wbb/example/main.go

```diff
@@ -0,0 +1,375 @@
+/*
+ * Copyright (c) 2022 Cisco Systems, Inc. and its affiliates
+ * All rights reserved.
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package main
+
+import (
+	"flag"
+	"github.com/cisco-open/go-p4/p4rt_client"
+	"github.com/cisco-open/go-p4/utils"
+	"github.com/golang/glog"
+	"github.com/openconfig/featureprofiles/feature/experimental/p4rt/wbb"
+	p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"
+	codes "google.golang.org/grpc/codes"
+	"net"
+	"os"
+	"time"
+)
+
+// Command line args
+var (
+	serverIP   = flag.String("server_ip", "192.0.2.1", "P4RT Server IP")
+	serverPort = flag.Int("server_port", 57400, "P4RT Server Port")
+	jsonFile   = flag.String("json_file", "./feature/experimental/p4rt/wbb/example/json/example.json", "JSON Params File Path")
+)
+
+func validateArgs() {
+	// Validate the IP
+	ip := net.ParseIP(*serverIP)
+	if ip == nil {
+		glog.Fatalf("Invalid Server IP: %s", *serverIP)
+	}
+}
+
+//
```

Comment:
Lines #50 and #52 can be removed.

Reply:
yes. this is from the other PR https://github.com/openconfig/featureprofiles/pull/490 in review. Since this PacketIn test only uses the wbb.go in that PR, maybe I can delete the tracking for that example folder to avoid those check failures. Reda will help address those in that PR.